components/common/ui/typography.js

```diff
@@ -0,0 +1,3 @@
+import typography from './typography.module.scss';
```

Comment:
does it make sense to create js file for this? why not to import scss module directly to places where we need it

Reply:
need to check, but I think in case of 2 imports it can be 2 different generated class modules